transaction/std/src/transaction_builder.rs

```diff
@@ -212,12 +208,12 @@ impl<FPR: FogPubkeyResolver> TransactionBuilder<FPR> {
     /// * `fog_hint_address` - The public address used to create the fog hint
     /// * `memo_fn` - The memo function to use (see TxOut::new_with_memo)
     /// * `rng` - RNG used to generate blinding for commitment
-    pub fn add_output_with_fog_hint_address<RNG: CryptoRng + RngCore>(
+    fn add_output_with_fog_hint_address<RNG: CryptoRng + RngCore>(
```

Comment:
If this is no longer being called from source, it should also get marked with `#[deprecated]`.

If things still do call it, then that will break the build, unfortunately, so it's tricky. When things are broken up into crates, it will be easier (IMO) to move the `-D warnings` stuff into CI and start being more expressive with warnings.

Reply:
i think it is being called from source, it's not actually deprecated

Reply:
it's being used at line 181 above, it is called by `add_change_output`